src/app/App.tsx

```diff
@@ -26,13 +27,25 @@ const LibraryStackScreen = () => {
     headerBackground: () => <Image style={styles.imageHeader} source={require('./assets/bc_nav_bar.png')} />,
     headerStyle: styles.headerStyle,
     cardStyle: styles.headerCard,
-    headerTitleStyle: styles.headerTitle,
-    headerLeft: () => <LogoutHeader />
+    headerTitleStyle: styles.headerTitle
   };
   return (
     <LibraryStack.Navigator initialRouteName="Library" screenOptions={headerOptions}>
-      <LibraryStack.Screen name="Library" component={Library} />
+      <LibraryStack.Screen
+        name="Library"
+        component={Library}
+        options={{ headerLeft: () => <LogoutHeader />, headerRight: () => <SearchHeader extended={false} /> }}
```

Comment:
the default value should always be false

Reply:
delete the prop

Reply:
Done! Completely changed